newsletter_automation/newsletter/routes.py

```diff
@@ -88,12 +89,13 @@ def logout():
         app.logger.error(e)
     return redirect('/')
 
-
 @app.route('/home')
 @app.route('/')
 @Authentication_Required.requires_auth
 def index():
-    return render_template('home.html', title="Home")
+    time.sleep(6)
```

Comment:
Why is there a sleep here I am guessing you forgot to delete the line after debugging?

Reply:
Yes I forgot to remove it after debugging. Removed it now